maint-lib/stage.py

```diff
@@ -49,6 +50,8 @@ def main(CORE_FILES_DIR, CEPH_RELEASES_DIR):
     logging.info('\n\n\n')  # Make it easier to determine where new runs start
     logging.info('Start time: {}'.format(time.ctime()))
 
+    exportCephVersionSpecifics()
```

Comment:
Should go after `verifyRequiredEnvVars()`.

Also, since still reworking, following the pattern set by GoArch and GitInfo, this should be renamed to something like `exportCephVersionSpecificEnvVars()`